src/Core/protected/Topo/TopoManager.h

```diff
@@ -103,6 +103,30 @@ class TopoManager: public Topo::TopoManagerIfc {
 	/** Réinitialisation     */
 	virtual void clear();
 
+    /*------------------------------------------------------------------------*/
+    /** \brief Création d'un sommet topologique à partir de coordonnées
+     *
+     *  \param pt le point contenant les coordonnées
+     *  \param groupName le nom du groupe auquel est associée ou non l'entité créée
+     */
+    virtual Mgx3D::Internal::M3DCommandResultIfc*
+    newTopoVertex(Utils::Math::Point pt, std::string groupName);
+
+
+    /*------------------------------------------------------------------------*/
+    /** \brief Création d'une entité topologique à partir de sommets existants
+     *
+     *  \param dim la dimension de l'entité à créer
+     *  \param ve le nom des sommet topologiques qui permettent de créer l'entité
+     *  \param groupName le nom du groupe auquel est associée ou non l'entité créée
+     */
+    virtual Mgx3D::Internal::M3DCommandResultIfc*
+    newTopoEntity(std::vector<std::string> ve, std::string groupName);
+
+    virtual Mgx3D::Internal::M3DCommandResultIfc*
+    newTopoEntity(std::vector<Vertex*> ve, std::string groupName);
```

Comment:
```suggestion
    virtual Mgx3D::Internal::M3DCommandResultIfc*
    newTopoEntity(std::vector<std::string>& ve, std::string groupName);

    virtual Mgx3D::Internal::M3DCommandResultIfc*
    newTopoEntity(std::vector<Topo::Vertex*>& ve, std::string groupName);
```
- pb with swig
- I think adding `Topo::` is not necessary but it might help avoid confusion with the `Geom::` one.